sympy/geometry/point.py

```diff
@@ -106,7 +106,7 @@ def __new__(cls, *args, **kwargs):
     is_Point = True
 
     def __contains__(self, item):
-        return item == self
+        return item in self.args
```

Comment:
ok. I'm +.01-- since Point is iterable itmakessense. I'm not sure if Point should be iterable.